cmd/template/cluster/flag.go

```diff
@@ -139,8 +141,18 @@ func (f *flag) Validate() error {
 		}
 	}
 
-	if f.Owner == "" {
-		return microerror.Maskf(invalidFlagError, "--%s must not be empty", flagOwner)
+	// TODO: Remove the flag completely some time after December 2021
+	if f.Owner == "" && f.Organization == "" {
+		return microerror.Maskf(invalidFlagError, "--%s or --%s must not be empty", flagOwner, flagOrganization)
```

Comment:
Let's use the desired flag name only here

```suggestion
		return microerror.Maskf(invalidFlagError, "--%s must not be empty", flagOrganization)
```